src/src/beneficiaries/beneficiaries.service.ts

```diff
@@ -2077,4 +2135,92 @@ export class BeneficiariesService {
 			data: result,
 		};
 	}
+
+	public async getAllDuplicatesUnderIp(id: number) {
```

Comment:
internal methods shall be private

Reply:
We won't be able to call it in controller of we make it private